website/static/js/addProjectPlugin.js

```diff
@@ -35,6 +34,9 @@ var AddProject = {
         self.newProjectDesc = m.prop('');
         self.newProjectCategory = m.prop(self.defaultCat);
         self.newProjectTemplate = m.prop('');
+        self.newProjectInstitutions = window.contextVars.currentUser.institutions.map(
```

Comment:
I went with contextVars because addProject is instantiated in multiple places, all of which would require pointing to the same piece of contextVars.